Templates/Databases/MicrosoftSQL/CSV file to MS SQL Database insert/long-description.md

```diff
@@ -0,0 +1,29 @@
+This process will read a **CSV file** which contains key-value pairs representing the data to be insert into the database from an **SFTP server**. 
+
+
+
+![Template](assets/CSV file to MS SQL Database insert.svg)
+
+# Prerequisites
+
+This template assumes that the following prerequisites are in place:
+
+- The SFTP server user should have the permissions to connect and access 
+  the files that Frends needs to download.
+- The Frends agent has access to database where the data will be insert and the necessary permissions to perform the insert.
+
+# Implementation and Usage Notes
+
+This template only performs inserts into the database specified in the Process Variables.
+The input CSV file will contained the data to be inserted in key-value format and must only contain data for one record.
```

Comment:
Fix typos please and we should allow for more that one row of data really. Inserting one row only makes this template quite useless.